crates/spirv-builder/src/lib.rs

```diff
@@ -204,7 +204,7 @@ fn invoke_rustc(builder: &SpirvBuilder, multimodule: bool) -> Result<PathBuf, Sp
         .unwrap_or_default();
 
     let rustflags = format!(
-        "-Z codegen-backend={} -Z symbol-mangling-version=v0{}",
+        "-Z codegen-backend={} -Zsymbol-mangling-version=v0 {}",
```

Comment:
Nit: This will cause a double space if `llvm_args` has a value, and a trailing space if `llvm_args` is empty. How it was before was correct.